website/docs/r/logs_index_order.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "datadog"
+page_title: "Datadog: datadog_logs_index_order"
+sidebar_current: "docs-datadog-resource-logs-index-order"
+description: |-
+  Provides a Datadog logs index order resource. This can be used to manage the order of logs indexes.
+---
+
+# datadog_logs_index_order
+
+Provides a Datadog [Logs Index API](https://docs.datadoghq.com/api/?lang=python#logs-indexes) resource. This can be used to manage the order of Datadog logs indexes.
+
+## Example Usage
+
+Create a Datadog logs pipeline order resource
```

Comment:
Can we put this as a comment in the sample code rather than up here?

Reply:
I was trying to make it look similar to `logs_pipeline`. Do you think I should update both of them (pipeline and index)?

Reply:
Hmm might be best to update both!

Reply:
Yup, let's either put it in as a comment in the sample code or just remove it altogether (I don't think it's adding too much value TBH).

Reply:
Sounds good. I will remove this line for both index_order and pipeline_order.